global_helpers/panther_oss_helpers.py

```diff
@@ -235,6 +237,32 @@ def set_key_expiration(key: str, epoch_seconds: int) -> None:
     )
 
 
+def put_dictionary(key: str, val: dict, epoch_seconds: int = None):
+    # Convert the data dictionary to a JSON string
+    data = json.dumps(val)
+
+    # Store the item in DynamoDB
+    kv_table().put_item(Item={"key": key, "data": {"S": data}})
+
+    if epoch_seconds:
+        set_key_expiration(key, epoch_seconds)
+
+
+def get_dictionary(key: str) -> dict:
+    # Retrieve the item from DynamoDB
+    response = kv_table().get_item(Key={"key": key})
+
+    # Check if the item was found
+    if "Item" in response:
+        item = response.get("Item", {})
+
+        # Deserialize the 'data' attribute back to a Python dictionary
+        data = json.loads(item.get("data", {}).get("S", {}))
+        return data
+
+    return {}
```

Comment:
```suggestion
    item = response.get("Item")
    if item is None:
        return {}
   
    # Deserialize the 'data' attribute back to a Python dictionary
    try:
       data = json.loads(item.get("data", {}).get("S", {}))
    except ValueError:
       data = {}
    return data
```

Reply:
it'd probably be better to raise.. I dunno... duo helpers uses the JSONDecodeError, which seems worthwhile.  https://github.com/panther-labs/panther-analysis/blob/master/global_helpers/panther_duo_helpers.py#L17-L18

Reply:
Raised!